src/pages/umi/web3js-differences-and-adapters.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Web3.js Differences and Adapters
+metaTitle: Umi - Web3.js Differences and Adapters
+description: Difference and Adapters to make Metaplex Umi work with Solana web3js.
+---
+
+The `@solana/web3.js` library is currently widely used in the Solana ecosystem and defines its own types for `Publickeys`, `Transactions`, `Instructions`, etc. 
+
+When creating `Umi`, we wanted to move away from the class-based types defined in `@solana/web3.js` and instead use a more functional approach by relying only on TypeScript types. This unfortunately means that, although having the same or similar import names, not all types from `@solana/web3.js` are compatible with the ones provided by `Umi` and vice versa.
+
+To help with this issue, `Umi` provides a set of adapters that allows to parse types to and from their `Web3.js` counterparts and they can be found in the [`@metaplex-foundation/umi-web3js-adapters`](https://www.npmjs.com/package/@metaplex-foundation/umi-web3js-adapters) package.
+
+## Required Package and Imports
+
+To install the `umi-web3js-adapters` package and be abel to access to a bunch of helper methods to convert to and from Web3.js types, use the following commands:
+
+```
+npm i @metaplex-foundation/umi-web3js-adapters
+```
+
+And this are all the imports that you will gain access to
+
+```ts
+import { 
+  // Instructions
+  fromWeb3JsInstruction, toWeb3JsInstruction,
+  // Keypairs
+  fromWeb3JsKeypair, toWeb3JsKeypair,
+  // Publickey
+  fromWeb3JsPublicKey, toWeb3JsPublicKey,
+  // Legacy Transactions
+  fromWeb3JsLegacyTransaction, toWeb3JsLegacyTransaction,
+  // Versioned Transactions
+  fromWeb3JsTransaction, toWeb3JsTransaction, 
+  // Messages
+  fromWeb3JsMessage, toWeb3JsMessage, toWeb3JsMessageFromInput
+} from '@metaplex-foundation/umi-web3js-adapters';
+```
+
+## Publickeys
+
+Generating public keys might seem similar at first sight, but there are some subtle differences between the packages. Web3Js uses a capital `P` and requires `new`, while the Umi version uses a lowercase `p`.
```

Comment:
we should stick to one notation. You have Publickeys in the header, public keys in the text